extensions/amp-truncate-text/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
This standalone Preact version does not work. I believe it's because the root component is missing certain styles that `amp-truncate-text` has, but haven't narrowed it down yet.

Reply:
I'd say try it with JSS.

Reply:
Done in local; will push soon